Collection/VideoCollection.js

```diff
@@ -13,28 +13,17 @@ var VideoCollection = Backbone.Collection.extend ({
     //????????????????????????????????change surfing to " +search+"??????????????????????????????????????????
     url: "https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=5&q=surfing&type=video&key=AIzaSyCx1sNMmT0WK3AmUp3UTLMPXX1v8Vw7Cqs",
       
-    //parse data from API to return without metadata
+    //parse data from API to return [] without metadata
     parse: function(response){
         var items  =  response.items;
         console.log("items: " + items);
```

Comment:
just a tip, when you're trying to console.log json response objects in the console, you'll see them as `[object Object]` because the console essentially doesn't know how to handle this type of data. So, if you really wanted to see what the items were you could say `console.log("items: " + JSON.stringify(items)`